### PR TITLE
cucumber framework and tests for Step 1

### PR DESCRIPTION
**Pull request by AlexaVB**
JIRA https://triadmoj.atlassian.net/browse/RST-270
URL: https://helpwithcourtfees.service.gov.uk/questions/form_name?locale=en
Cucumber framework for HwF
Tests for Step 1

### CODE REVIEW

**Comment by zaparka** [comment]
Looks good except that ```# frozen_string_literal: true``` bit.

File: features/step_definitions/step_one_steps.rb [hunk 1]
```diff
@@ -16,20 +16,20 @@
 end
 
 When(/^I select I don’t know the form name or number$/) do
-  form_name = step_one_page.content.new_form_name
-  expect(form_name.form_group[1].block_label[0].text)
-    .to eq 'I don’t know the form name or number'
+  form_name = step_one_page.content.new_form_name eq 'I don’t know the form name or number'
```
Comment (AlexaVB) [inline features/step_definitions/step_one_steps.rb]: This won't work.  I have reverted this line.

**Comment by leoapost** [comment]
@zaparka good catch. I've pushed clean up commits.

**Comment by leoapost** [comment]
@AlexaVB good catch! Should be fine now.

**Comment by zaparka** [comment]
The code climate is still broken.

**Pull request merged**